packages/desktop-client/src/components/table.tsx

```diff
@@ -242,27 +263,24 @@ export function Cell({
           // the user does a direct click, not if they also drag the
           // mouse to select something
           onMouseDown={e => (mouseCoords.current = [e.clientX, e.clientY])}
-          onMouseUp={e => {
-            if (
-              mouseCoords.current &&
-              Math.abs(e.clientX - mouseCoords.current[0]) < 5 &&
-              Math.abs(e.clientY - mouseCoords.current[1]) < 5
-            ) {
-              onExpose && onExpose(name);
-            }
-          }}
           // When testing, allow the click handler to be used instead
-          onClick={global.IS_TESTING && (() => onExpose && onExpose(name))}
+          onClick={
+            global.IS_TESTING
+              ? () => onExpose && onExpose(name)
+              : e => {
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I think this is breaking the `tab` behavior in the table. When you click `tab` - it clears the payee field and throws an exception.